Emby.Server.Implementations/ScheduledTasks/TaskManager.cs

```diff
@@ -38,7 +38,7 @@ public TaskManager(
             _applicationPaths = applicationPaths;
             _logger = logger;
 
-            ScheduledTasks = Array.Empty<IScheduledTaskWorker>();
+            ScheduledTasks = [];
```

Comment:
this will create a new instance each time, please use the Array.Empty which reuses the instances instead.

Reply:
This is not true, `[]` gets optimized to the same assembly as Array.Empty
https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunHjyADAG0AugwCyACgCUHLjwXEA7AwB2MAO5NBQgSIDc87gF8jDM/W3Cx40jLnUFilQEEoUbAE8AdAFF8AA4YngA8/AIAfDKGjjymsdwWfDo2dPacCU7KDKIxCvHGQA===

Reply:
In that case, disregard